backend/controller/ml_model/compiler.py

```diff
@@ -81,77 +86,96 @@ class UniqueId(shared.StrEnum):
 
 
 class VariableRole(shared.StrEnum):
-  FEATURE = 'FEATURE',
-  LABEL = 'LABEL',
-  TRIGGER_EVENT = 'TRIGGER_EVENT',
-  FIRST_VALUE = 'FIRST_VALUE',
-  TRIGGER_DATE = 'TRIGGER_DATE',
-  USER_ID = 'USER_ID',
+  FEATURE = 'FEATURE'
+  LABEL = 'LABEL'
+  TRIGGER_EVENT = 'TRIGGER_EVENT'
+  FIRST_VALUE = 'FIRST_VALUE'
+  TRIGGER_DATE = 'TRIGGER_DATE'
+  USER_ID = 'USER_ID'
   CLIENT_ID = 'CLIENT_ID'
+  GCLID = 'GCLID'
 
 
 class VariableSet():
-  source: Source
-  input: models.MlModelInput
-  features: list[models.MlModelVariable]
-  label: models.MlModelVariable
-  trigger_event: models.MlModelVariable
-  first_value: models.MlModelVariable
-  _trigger_date: models.MlModelVariable
-  _unique_id: str
-  user_id: models.MlModelVariable
-  client_id: models.MlModelVariable
-
-  def __init__(self, source: Source, input: models.MlModelInput, unique_id: UniqueId) -> None:
-    self.source = source
-    self.input = input
-    self.features = []
-    self.label = None
-    self.trigger_event = None
-    self.first_value = None
-    self._trigger_date = None
-    self._unique_id = unique_id.lower()
-    self.user_id = 'user_id'
-    self.client_id = 'user_pseudo_id'
+  _items: list[models.MlModelVariable]
```

Comment:
```_items``` is a generic variable name, how about making it more descriptive (ie, ```_model_variables```).

Reply:
It's the items in a VariableSet. I thought it somewhat redundant at first, but perhaps it's confusing. I've changed the name to _variables for clarity.